variables.data_lake.tf

```diff
@@ -45,3 +45,48 @@ EOT
     error_message = "Key `legacy` is preserved for backward compatibility."
   }
 }
+
+variable "storage_data_lake_gen2_paths" {
```

Comment:
We can add `nullable = false` in this `variable` to simplify the `for_each` expression.